core/src/stages/move_relative.cpp

```diff
@@ -286,8 +294,14 @@ bool MoveRelative::compute(const InterfaceState& state, planning_scene::Planning
 		// offset from link to ik_frame
 		const Eigen::Isometry3d& offset = scene->getCurrentState().getGlobalLinkTransform(link).inverse() * ik_pose_world;
 
-		success =
+		const auto planner_solution_maybe =
 		    planner_->plan(state.scene(), *link, offset, target_eigen, jmg, timeout, robot_trajectory, path_constraints);
+		if (planner_solution_maybe.has_value()) {
+			success = planner_solution_maybe.value();
+		}
+		if (!success) {
+			error_message = planner_solution_maybe.error();
```

Comment:
Where does the error message come from? Isn't this overwriting a previous error message?